Formula/v/vamp-plugin-sdk.rb

```diff
@@ -5,6 +5,7 @@ class VampPluginSdk < Formula
   url "https://deb.debian.org/debian/pool/main/v/vamp-plugin-sdk/vamp-plugin-sdk_2.10.0.orig.tar.gz"
   mirror "https://code.soundsoftware.ac.uk/attachments/download/2691/vamp-plugin-sdk-2.10.0.tar.gz"
   sha256 "aeaf3762a44b148cebb10cde82f577317ffc9df2720e5445c3df85f3739ff75f"
+  license all_of: ["MIT", "BSD-3-Clause"]
```

Comment:
Actually might be closer to "X11" than "MIT" due to no-advertising clause?
https://github.com/vamp-plugins/vamp-plugin-sdk/blob/vamp-plugin-sdk-v2.10/COPYING#L25-L29
```
    Except as contained in this notice, the names of the Centre for
    Digital Music; Queen Mary, University of London; and Chris Cannam
    shall not be used in advertising or otherwise to promote the sale,
    use or other dealings in this Software without prior written
    authorization.
```

https://spdx.org/licenses/X11.html
```
Notes
  This is same as MIT, but with no advertising clause added.

...

Except as contained in this notice, the name of the <X Consortium> shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization from the <X Consortium>.
```
The `<X Consortium>` is in red so substitutable - https://spdx.github.io/spdx-spec/v2.3/license-matching-guidelines-and-templates/#b34-guideline-replaceable-text